### ISSUE TITLE
Crash when attempting to change username during signup

### ISSUE THREAD

**Issue by rachelmcr** [wordpress-mobile/WordPress-iOS#11444]
### Expected behavior

I expect to be able to change my username during the signup flow.

### Actual behavior

When I tap on the username field on the signup epilogue screen, the app crashes.

```
Fatal Exception: NSInternalInconsistencyException
0  CoreFoundation                 0x197023758 __exceptionPreprocess
1  libobjc.A.dylib                0x19622bd00 objc_exception_throw
2  CoreFoundation                 0x196f298b4 -[NSCache init]
3  UIKitCore                      0x1c3ba8fe0 -[UINib instantiateWithOwner:options:]
4  UIKitCore                      0x1c41123d8 -[UITableView _dequeueReusableViewOfType:withIdentifier:]
5  WordPress                      0x10332b7a8 SignupUsernameTableViewController.searchFieldCell() (SignupUsernameTableViewController.swift:188)
6  WordPress                      0x10332ca64 specialized SignupUsernameTableViewController.tableView(_:cellForRowAt:) (SignupUsernameTableViewController.swift:130)
7  WordPress                      0x10332b9b8 @objc SignupUsernameTableViewController.tableView(_:cellForRowAt:) (<compiler-generated>)
8  UIKitCore                      0x1c412c188 -[UITableView _createPreparedCellForGlobalRow:withIndexPath:willDisplay:]
9  UIKitCore                      0x1c412c6a8 -[UITableView _createPreparedCellForGlobalRow:willDisplay:]
10 UIKitCore                      0x1c40f7fbc -[UITableView _updateVisibleCellsNow:isRecursive:]
11 UIKitCore                      0x1c4115b7c -[UITableView layoutSubviews]
12 UIKitCore                      0x1c43a9ddc -[UIView(CALayerDelegate) layoutSublayersOfLayer:]
13 libobjc.A.dylib                0x196227454 -[NSObject performSelector:withObject:]
14 QuartzCore                     0x19b546488 -[CALayer layoutSublayers]
15 QuartzCore                     0x19b546774 CA::Layer::layout_if_needed(CA::Transaction*)
16 QuartzCore                     0x19b4ae28c CA::Context::commit_transaction(CA::Transaction*)
17 QuartzCore                     0x19b4dcfbc CA::Transaction::commit()
18 QuartzCore                     0x19b4ddd48 CA::Transaction::observer_callback(__CFRunLoopObserver*, unsigned long, void*)
19 CoreFoundation                 0x196fb2ee8 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__
20 CoreFoundation                 0x196fad9e0 __CFRunLoopDoObservers
21 CoreFoundation                 0x196fadf40 __CFRunLoopRun
22 CoreFoundation                 0x196fad764 CFRunLoopRunSpecific
23 GraphicsServices               0x1991e9dd4 GSEventRunModal
24 UIKitCore                      0x1c3f0400c UIApplicationMain
25 WordPress                      0x102f22d44 main (main.m:11)
26 libdyld.dylib                  0x196a6cffc start
```

5cadf1e1f8b88c296356b8b3-fabric (WP Internal)

### Steps to reproduce the behavior

1. Open the app logged out.
2. Start the email signup flow.
3. Enter a new email address and confirm the signup via the magic link.
4. On the signup epilogue screen, tap the username field to try to change your username.

Result: Crash.

##### Also tested on 12.1 app store version and could not reproduce there.

**Comment by rachelmcr** [comment]
cc @jkmassel — I don't see any recent changes to this file, but this seems to be a new crash in the 12.2 beta.

**Comment by jkmassel** [comment]
cc @mindgraffiti – could this have anything to do with the login library changes? 🤔

**Comment by mindgraffiti** [comment]
@rachelmcr @jkmassel yes, it's quite possibly the Authenticator changes I made. I'm sorry I didn't communicate that well. They were breaking changes that are medium-risk, including parameter type changes to the existing delegate methods and changing the Credentials to a struct. I asked @astralbodies if he would investigate this one, since I'll be leaving shortly to give a talk.

**Comment by astralbodies** [comment]
I'm on it 😄

**Comment by astralbodies** [comment]
Dug into a bit and it looks like the recent changes to the WordPressAuthenticator pod are the culprit. We moved from a dynamic framework to a static library which changed where on disk the bundle lives for the resources like XIBs. This crash is due to the XIB `SearchTableViewCell` not being found.

Talked with @jtreanor and he's going to do the quick fix to address this in WPiOS. 

I found three references needing to be updated:

<img width="374" alt="2019-04-10_14-06-36" src="https://user-images.githubusercontent.com/373903/55906427-dc8f1800-5b99-11e9-84f7-8c1098a6750a.png">
